distribution/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# https://hub.docker.com/_/openjdk
+FROM centos:7.5.1804
+
+# set label
+LABEL maintainer="seata <niao.shuai123@163.com>"
+
+# set environment
+ENV SEATA_USER="seata" \
+    JAVA_HOME="/usr/lib/jvm/java-1.8.0-openjdk" \
+    JAVA="/usr/lib/jvm/java-1.8.0-openjdk/bin/java" \
+    TIME_ZONE="Asia/Shanghai" 
+
+ARG SEATA_VERSION=0.9.0
+
+WORKDIR /$BASE_DIR
+
+RUN set -x \
+    && yum update -y \
+    && yum install -y java-1.8.0-openjdk java-1.8.0-openjdk-devel wget iputils nc vim libcurl \
+    && wget  https://github.com/seata/seata/releases/download/v${SEATA_VERSION}/seata-server-${SEATA_VERSION}.tar.gz -P /home \
```

Comment:
Since the image is made in project build, it should't download from RELEASE, please use the distribution directory.